packages/library-sync/build.gradle.kts

```diff
@@ -51,9 +51,7 @@ kotlin {
                 // Cinterop does not hold anything required by users
                 implementation(project(":cinterop"))
 
-                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:${Versions.coroutines}") {
-                    version { strictly(Versions.coroutines) }
```

Comment:
We can use the [range notation](https://docs.gradle.org/current/userguide/single_versions.html#simple_version_declaration_semantics) to declare the minimum version we support which uses the new memory model ...